android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt

```diff
@@ -163,23 +206,27 @@ class GeoWidgetFragment : Fragment() {
     private fun setOnClickLocationListener(mapView: KujakuMapView) {
         mapView.setOnFeatureClickListener(
             { featuresList ->
-                val feature = featuresList.firstOrNull() ?: return@setOnFeatureClickListener
-                if (feature.geometry() !is Point) {
+                val mapBoxFeature = featuresList.firstOrNull() ?: return@setOnFeatureClickListener
+                if (mapBoxFeature.geometry() !is Point) {
```

Comment:
Leave a TODO here. This will be refactored in future to support other types not just `Point`